prometheus-to-sd/main.go

```diff
@@ -143,7 +144,7 @@ func readAndPushDataToStackdriver(stackdriverService *v3.Service, gceConf *confi
 		}
 		metrics, err := translator.GetPrometheusMetrics(sourceConfig.Host, sourceConfig.Port)
 		if err != nil {
-			glog.Warningf("Error while getting Prometheus metrics %v", err)
+			glog.Warningf("Error while getting Prometheus metrics %v for component %v", err, sourceConfig.Component)
```

Comment:
Please use V(2), since it's in a loop. If e.g. a container was restarted couple of times and there's a 5 min backoff, logs will be flooded